.pre-commit-config.yaml

```diff
@@ -9,4 +9,4 @@ repos:
     rev: 23.12.1
     hooks:
     -   id: black
-        files: ^src/
+        files: ^src/|^test/
```

Comment:
This was missing @alejoe91 you can merge the other one and then this.

Reply:
right!